build/Dependencies.props

```diff
@@ -42,6 +42,7 @@
   <!-- Test-only Dependencies -->
   <PropertyGroup>
     <BenchmarkDotNetVersion>0.11.3</BenchmarkDotNetVersion>
+    <MicrosoftCodeAnalysisTestingVersion>1.0.0-beta1-63812-02</MicrosoftCodeAnalysisTestingVersion>
```

Comment:
I don't know anything regarding analyzers. Is it normal to use beta package? Is where are stable version available?

Reply:
This is a dependency we control. We're still stabilizing the API for the testing package (and determining whether breaking changes are allowed in updates after stable) so this is what's currently available. The package performs *much* more rigorous testing than previous harnesses, which will show up in some of the comments.